src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx

```diff
@@ -30,11 +29,6 @@ class DisabilityBenefitsApp extends React.Component {
         loginUrl={this.props.loginUrl}
         verifyUrl={this.props.verifyUrl}>
         <AppContent>
-          <div>
-            <div className="row">
-              <Breadcrumbs location={this.props.location}/>
```

Comment:
Why was this removed?

Reply:
It looks like it's included in the children now instead.

Reply:
The breadcrumbs are now either in specific page components (claims) or in the AppealsLayout (appeals).